lib/apparmortest.pm

```diff
@@ -23,7 +23,7 @@ use warnings;
 use testapi;
 use utils;
 use version_utils qw(is_sle is_leap is_tumbleweed);
-use y2x11test qw(launch_yast2_module_x11);
+use y2_guitest qw(launch_yast2_module_x11);
```

Comment:
As we discussed, it would be great to somehow identify in the file name if this is related to installation or to YaST modules testing. Currently it looks like YaST2 GUI test, which does not tell to test developer what this perl module is responsible for.

Reply:
@OleksandrOrlov i have renamed both y2_consoletest and y2_guitest to y2_module_*.